### ISSUE TITLE
Accessibility Suggestion - Consider making this clickable item larger (for Android)

### ISSUE THREAD

**Issue by PrinceSP** [roto93/react-native-neat-date-picker#38]
These item's size is `34dp x 34dp` and `24dp x 44dp`. Consider making this **Touch Target** 48dp wide and 48dp high or larger.

<img width="232" alt="Screen Shot 2024-10-16 at 15 09 09" src="https://github.com/user-attachments/assets/c7b26f82-4089-40ec-bab9-7105e3d9078e">

<img width="238" alt="Screen Shot 2024-10-16 at 15 11 49" src="https://github.com/user-attachments/assets/47116aa5-9e13-4873-a410-e57574cad7f7">

Any on-screen element that someone can click, touch, or otherwise interact with should be large enough for reliable interaction. Consider making sure these elements have a width and height of at least 48dp, as described in the [Material Design Accessibility guidelines](https://material.google.com/usability/accessibility.html#accessibility-layout), and also check this new documentaiton for best practice in android [Icon Button Specs](https://m3.material.io/components/icon-buttons/specs).

**Comment by roto93** [comment]
Thank you for the suggestion.
Please try the version above `v15.0`
